### PR TITLE
Change some outputs to stdout

### PR DESCRIPTION
**Pull request by aanil**
Changed some outputs to stdout so that it can be more easily parsed when dds is called from another program.

- Used the `--no-prompt` option to decide between rich and json outputs when running `dds put`.

### CODE REVIEW

**Review by inaod568 — COMMENTED** [review]
Apart from my comment, I've created the following card:
https://github.com/scilifelabdatacentre/dds_web/issues/896

File: dds_cli/data_remover.py [hunk 1]
```diff
@@ -62,30 +63,36 @@ def __create_failed_table(self, resp_json, level="File"):
...
-
-            # Add rows
-            for x in not_exists:
-                table.add_row(x, f"No such {level.lower()}")
-
-            for x, y in delete_failed.items():
-                table.add_row(
-                    f"[light_salmon3]{x}[/light_salmon3]",
-                    f"[light_salmon3]{y}[/light_salmon3]",
+            if not self.no_prompt:
```
[the remark this answers is not in the document]
Comment (aanil) [reply dds_cli/data_remover.py]: Done!

**Comment by aanil** [comment]
Addressed Issue https://github.com/ScilifelabDataCentre/dds_cli/issues/239 as well

File: dds_cli/__main__.py [hunk 2]
```diff
@@ -699,12 +694,12 @@ def create(
...
             if created:
-                LOG.info(
+                dds_cli.utils.console.print(
                     f"Project created with id: {project_id}",
                 )
                 if user_addition_messages:
                     for msg in user_addition_messages:
-                        LOG.info(msg)
+                        dds_cli.utils.console.print(msg)
                         LOG.info(
```
Comment (inaod568) [inline dds_cli/__main__.py:703]: Can you change here too?

Comment (aanil) [reply dds_cli/__main__.py]: Ah missed one, done!

**Review by inaod568 — APPROVED** [review]

**Comment by inaod568** [comment]
As you said during the meeting @aanil I think these changes are fine and we can change more after testing if we get comments about that.

**Pull request merged**